frontend/src/container/CreateAlertRule/SelectAlertType/styles.ts

```diff
@@ -0,0 +1,22 @@
+import { Card, Row } from 'antd';
+import styled from 'styled-components';
+
+export const SelectTypeContainer = styled.div`
+	&&& {
+		padding: 1rem;
+	}
+`;
+
+export const AlertTypeCards = styled(Row)`
+	&&& {
+		flex-wrap: nowrap;
+	}
+`;
+
+export const AlertTypeCard = styled(Card)`
+	&&& {
+		margin: 5px;
+		width: 300px;
```

Comment:
do really need `300px` if yes please convert it to `18.75rem`

Reply:
done.